src/main/resources/application.yml

```diff
@@ -1,3 +1,18 @@
 micronaut:
   application:
     name: core
+  router:
+    static-resources:
+      swagger:
+        paths: classpath:META-INF/swagger
+        mapping: /swagger/**
+      swagger-ui:
+        paths: classpath:META-INF/swagger/views/swagger-ui
+        mapping: /swagger-ui/**
+
+endpoints.all.sensitive: false
+
```

Comment:
Is it really need to have such many empty lines below? :)

Reply:
Of course!